packages/devtools_app/lib/src/screens/performance/panes/frame_analysis/frame_analysis.dart

```diff
@@ -48,6 +56,36 @@ class FlutterFrameAnalysisView extends StatelessWidget {
           Expanded(
             child: FrameTimeVisualizer(frameAnalysis: frameAnalysis),
           ),
+          ValueListenableBuilder<ServiceExtensionState>(
+            valueListenable:
+                serviceManager.serviceExtensionManager.getServiceExtensionState(
+              extensions.trackRebuildWidgets.extension,
+            ),
+            builder: (context, state, _) {
+              final rebuilds =
+                  rebuildCountModel.rebuildsForFrame(frameAnalysis.frame.id);
+              if (!state.enabled && rebuilds == null) {
+                // No need to explain why there are no rebuilds as rebuild
+                // tracking is disabled.
+                return const SizedBox();
+              }
+              if (rebuilds == null) {
+                return const Text(
+                    'Rebuild information not available for frame');
+              } else if (rebuilds.isEmpty) {
+                return const Text(
+                  'No rebuilds of widgets directly created by the project occurred.',
+                );
+              } else {
+                return Expanded(
```

Comment:
refactored this a bit. unfortunately this can't really go in a scrollview because the rebuild table also needs to scroll and making that all work well seems more trouble than it is worth. I've added a todo to worry about this if we get more frame hints.